march_joint_inertia_controller/CMakeLists.txt

```diff
@@ -1,26 +1,45 @@
 cmake_minimum_required(VERSION 3.0.2)
 project(march_joint_inertia_controller)
 
-add_compile_options(-std=c++14 -Wall -Wextra -Werror)
+add_compile_options(-std=c++17 -Wall -Wextra -Werror)
```

Comment:
Why did you upgrade to `c++17`. I think ROS Melodic only supports `c++14`

Reply:
I implemented Rutgers suggestion to use the std::clamp function. For the compiler to find this function I needed to upgrade to c++17 according to some stackoverflow page that I can't find anymore.

Reply:
Yes [`std::clamp`](https://en.cppreference.com/w/cpp/algorithm/clamp) is indeed c++17. However, I am not sure if everything will be working correctly, since everything else is c++14. Perhaps change it back to c++14 just to be sure if it's only for this method that can also be written with `std::min` and `std::max`?

Reply:
cool cool, I don't think changing this back affects the performance, so I think I ´ll change this back